packages/dialtone-vue3/components/popover/popover.vue

```diff
@@ -953,6 +968,48 @@ export default {
         onShow: this.onShow,
       });
     },
+
+    //  ============================================================================
+    //  $ HOVERCARD
+    //  ----------------------------------------------------------------------------
+
+    setInTimer () {
+      this.inTimer = setTimeout(() => {
+        this.isOpen = true;
+      }, TOOLTIP_DELAY_MS);
+    },
```

Comment:
Hmm... currently on popover with hovercard = false it will not set any timer, because those functions are called when hovercard is true.
When is false `inTimer: null,` and `outTimer: null,`.

Where `it will set a timer that never ends` ?

Reply:
oh my bad I thought this was a event handler lol. it's all good.